db/migrate/20220714204023_create_users.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :name
       t.string :photo
       t.text :bio
-      t.integer :postCounter
+      t.integer :postsCounter
```

Comment:
- [ ] [OPTIONAL] Ruby naming convention should be followed every time, the attribute `postsCounter` should read `post_counter`.